validator/keymanager/imported/keymanager.go

```diff
@@ -261,15 +264,24 @@ func (dr *Keymanager) DeleteAccounts(ctx context.Context, publicKeys [][]byte) e
 	return nil
 }
 
-// FetchValidatingPublicKeys fetches the list of public keys from the imported account keystores.
+// FetchValidatingPublicKeys fetches the list of active public keys from the imported account keystores.
```

Comment:
I decided to update this function as I'm assuming this could be the least impactful to the remaining of the codebase? 

Things to consider with the current stage of the PR (with modifying `FetchValidatingPublicKeys`):
- Listing accounts via RPC will not list disabled accounts
- Metrics `LogValidatorGainsAndLosses` will not take into account disabled accounts
- `accounts delete` will not list disabled accounts as interactive options
- `accounts backup` will not list disabled accounts as interactive options
- `accounts list` will not list disabled accounts

Let me know if those need to be resolved or not as well.

Reply:
- Listing accounts via RPC will not list disabled accounts
- Metrics LogValidatorGainsAndLosses will not take into account disabled accounts

These are fine ^, however, the others should indeed work, especially `list`. Perhaps we can have a function called FetchAllValidatingPublicKeys() which is used by list, delete, and backup?

Reply:
added as requested.